golang/x/relay/keeper/requests.go

```diff
@@ -121,7 +133,7 @@ func (k Keeper) getNextID(ctx sdk.Context) (types.RequestID, sdk.Error) {
 	store := k.getRequestStore(ctx)
 	idTag := []byte(types.RequestIDTag)
 	if !store.Has(idTag) {
-		store.Set(idTag, bytes.Repeat([]byte{0}, 8))
+		store.Set(idTag, bytes.Repeat([]byte{1}, 8))
```

Comment:
stared with 1, do avoid doing an extra 0 bytes check.

Reply:
```bytes.Repeat([]byte{1}, 8))```

will end up with `1` repeated 8 times. `[1, 1, 1, 1, 1, 1, 1, 1]` which corresponds to hex `0x0101010101010101`

If you want to start with 1 do 
```
next := make([]byte, 8)
next[7] = 1
```

I don't think you need to change this tho. Where is the extra 0 bytes check?

Reply:
Maybe this has to do with changing the datatype but if I have a 0 id, this is the error I get when handing in `0x`:
```
➜  golang git:(issue28) ✗ relaycli tx relay provideproof  1_check_proof.json 3_filled_requests.json  --from me  --inputfile --broadcast-mode block
ERROR: Expected 8 bytes, got 0 bytes
```

Reply:
hand in `0x0000000000000000`. It expects these to be 8-bytes long regardless of whether those bytes are 0